### ISSUE TITLE
Color Contrast Ratio not updated correctly

### ISSUE THREAD

**Issue by OLLI-S** [ThePacielloGroup/CCAe#277]
**Describe the bug**
I start CCA 3.2.0 portable and there are the default color values (#000000 and #FFFFFF) shown.

![image](https://user-images.githubusercontent.com/8037790/216256441-3e681a27-0bbd-4442-b5a3-4ee6fac7b8ed.png)

In a screenshot I select the foreground color with the color picker (color #FEFEFE), then the Color Contrast Ratio of 1:1 is shown (as expected):

![image](https://user-images.githubusercontent.com/8037790/216256874-6e928561-c9ca-4c73-bb0a-d7f72c430792.png)

Then I select the background color with the color picker (color #FE631E) and the Color Contrast Ratio of 1:1 is still shown (this is a **bug**):

![image](https://user-images.githubusercontent.com/8037790/216254816-ff5bc9b9-e703-40cb-95a1-9d25574a001f.png)

If I select afterwards a black background color (#000000) using the color picker, the Color Contrast Ratio of 20.8:1 is shown (as expected):

![image](https://user-images.githubusercontent.com/8037790/216257197-ea94ca8d-66f6-47e5-982f-67dd7bbfadbc.png)

If I select afterwards again the orange background color (#FE631E) then the Color Contrast Ratio of 20.8:1 is still shown (this is a **bug**):

![image](https://user-images.githubusercontent.com/8037790/216257328-d8067e99-3899-41ac-bcbe-6df95fca2bc8.png)

When I do the same with a darker orange background color (#D14520), then the Color Contrast Ratio of 4.5:1 is shown (what is correct):

![image](https://user-images.githubusercontent.com/8037790/216258848-87f601ba-750b-4c05-9feb-f3206d080e39.png)

The problem occurs only, when the background color is #FE631E.

**Desktop (please complete the following information):**
 - OS: Windows 10 **Enterprise** x64
 - Version: 21H2

**Comment by OLLI-S** [comment]
I downloaded the older version **CCA-Portable-3.1.4.exe** and here ist works as expected:

![image](https://user-images.githubusercontent.com/8037790/218476642-bd29be0d-fa82-4bcd-b655-119fb03cd00f.png)

**Comment by ferllings** [comment]
Fixed in 0e9a5cb8ccd63a9b80afe8c4000b79fe2bc4c777

**Comment by OLLI-S** [comment]
@ferllings Thank you for fixing this issue and releasing the new version so quickly!
Really appreciate that!

**Comment by ferllings** [comment]
I appreciate the 'so quickly' ;)
